src/libcommonserver/io/iohelper.h

```diff
@@ -88,7 +88,7 @@ struct IoHelper {
          dangling symlink is an existing item of file type), false otherwise. \param ioError holds the error returned when an
          underlying OS API call fails. \return true if no unexpected error occurred, false otherwise.
          */
-        static bool getFileStat(const SyncPath &path, FileStat *filestat, bool &exists, IoError &ioError) noexcept;
+        static bool getFileStat(const SyncPath &path, FileStat *filestat, IoError &ioError) noexcept;
```

Comment:
The docstring above needs to be adapted.

Reply:
Done